setup.py

```diff
@@ -20,7 +20,7 @@
     license=license,
     packages=find_packages(exclude=('tests', 'docs')),
     install_requires=[
-        'ethereum == 1.3.7',
+        'ethereum==2.0.4',
         'serpent',
```

Comment:
Yes, I think that's a best practice; though `serpent` here is incorrect (it should be `ethereum-serpent`) and testing requirements should not be in `install_requires`, they should actually be in `tests_require`.